.pre-commit-config.yaml

```
@@ -0,0 +1,8 @@
+repos:
+-   repo: https://github.com/yzhuang52/student-developer-setup
+    rev: 0.0.1
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+
```

In your pre-commit config the values of `repo` are repository URLs that have git-commit hooks defined in them for others to use, not the name of your local repo.

Check out https://pre-commit.com/#usage for more info, but you can probably just use the sample config generated by

```
pre-commit sample-config > .pre-commit-config.yaml
```

to get started

```suggestion
repos:
-   repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v3.2.0
    hooks:
    -   id: trailing-whitespace
    -   id: end-of-file-fixer
    -   id: check-yaml
    -   id: check-added-large-files
```